src/graphql/mutations/createProgram.graphql

```diff
@@ -0,0 +1,7 @@
+mutation createProgram($data: CreateProgramInput!) {
+  createProgram(data: $data) {
+    programId
+    createdAt
```

Comment:
Do we need to include createAt and updatedAt (isn't it autohandled by the backend)